File: Funciones/4. Calcular_nota/js/nota.js
/**
 * Calcular la nota
 * Autor: Santiago GM
 * 02/04/2024
*/

//con parametros 
let nota;
let porcentaje;

function calcularNota(pnota,pporcentaje){

    nota = pnota;
    porcentaje = pporcentaje;

    let resultado;
    resultado = nota * porcentaje;

    return `El porcentaje de ${nota} es: ${resultado}`;
    
}

const calcularNotaExp = function(pnota,pporcentaje){

    nota = pnota;
    porcentaje = pporcentaje;

    let resultado;
    resultado = nota * porcentaje;

    return `El porcentaje de ${nota} es: ${resultado}`;
    
}

File: python/4. Calcular_nota/utils/nota.py
# Calcular la nota
# Autor: Santiago GM

# Con función con parámetros
def calcular_notas(nota_uno_p, nota_dos_p, nota_tres_p):
    nota_uno, nota_dos, nota_tres = nota_uno_p, nota_dos_p, nota_tres_p
    promedio = (nota_uno + nota_dos + nota_tres) / 3
    return promedio

# Como función lambda
calcular_nota_Exp = lambda pnotaUno,pnotaDos,pnotaTres : (pnotaUno + pnotaDos + pnotaTres) / 3;